File: JingFM-RoundEffect/ViewController.m
//
//  ViewController.m
//  JingFM-RoundEffect
//
//  Created by isaced on 13-6-6.
//  Copyright (c) 2013年 isaced. All rights reserved.
//  By isaced:http://www.isaced.com/

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
    
    
    self.view.backgroundColor = [UIColor colorWithPatternImage:[UIImage imageNamed:@"background.jpg"]];
    
    self.roundView.delegate = self;
    self.roundView.roundImage = [UIImage imageNamed:@"girl"];
    self.roundView.rotationSpeed = 3.0;
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

-(void)playStatuUpdate:(BOOL)playState
{
    NSLog(@"%@...", playState ? @"Playing": @"Pause");
}

@end

File: JingFM-RoundEffect-Swift/ViewController.swift
//
//  ViewController.swift
//  JingFM-RoundEffect-Swift
//
//  Created by isaced on 14-6-11.
//  Copyright (c) 2014 Year isaced. All rights reserved.
//

import UIKit

class ViewController: UIViewController,JingRoundViewDelegate {
                            
    @IBOutlet var roundView : JingRoundView
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        self.view.backgroundColor = UIColor(patternImage: UIImage(named: "background.jpg"))
        self.roundView.roundImage = UIImage(named: "girl")
        self.roundView.isPlay = false;
        self.roundView.delegate = self
        
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    func playStatuUpdate(playState: Bool) {
        NSLog("%@...", playState ? "Playing": "Pause")
    }


}

